src/main/java/dev/openfeature/sdk/EventProvider.java

```diff
@@ -2,6 +2,8 @@
 
 import dev.openfeature.sdk.internal.TriConsumer;
 
+import javax.annotation.Nullable;
```

Comment:
We can't use javax, unfortunately. It's non-standard and creates a problematic dependecny for consumers (see https://github.com/open-feature/java-sdk/issues/867)